LibSWBF2/Chunks/BaseChunk.cpp

```diff
@@ -35,7 +35,9 @@ namespace LibSWBF2::Chunks
 			THROW("Invalid Chunk: '{}' Size: {:#x} At Position: {:#x} with File Size of: {:#x}", m_Header, m_Size, stream.GetPosition(), stream.GetFileSize());
 		}
 
-		if (stream.GetPosition() + m_Size > stream.GetFileSize())
+		//Before I added 8 to the file size, an exception was thrown at the initial
+		//ucfb chunk on geo1.lvl.  Not sure why this never happened for other maps...
+		if (stream.GetPosition() + m_Size > 8 + stream.GetFileSize())
 		{
 			THROW("Chunk is too big and will end up out of file! Chunk: '{}' Size: {:#x} At Position: {:#x} with File Size of: {:#x}", m_Header, m_Size, stream.GetPosition() - 8, stream.GetFileSize());
 		}
```

Comment:
Haven't dug into this yet, maybe it has something to do w/line ending conversions between OS's?

Reply:
Good catch, forgot to add it after altering `ThereIsAnother`